qiskit_transpiler_service/wrappers/base.py

```diff
@@ -180,18 +180,22 @@ def _raise_transpiler_error_and_log(msg: str):
 
 
 def _get_error_msg_from_response(exc: requests.exceptions.HTTPError):
-    resp = exc.response.json()
-    detail = resp.get("detail")
-    # Default message
-    msg = "Internal error."
-
-    if isinstance(detail, str):
-        msg = detail
-    elif isinstance(detail, list):
-        detail_input = detail[0]["input"]
-        detail_msg = detail[0]["msg"]
-
-        if detail_input and detail_msg:
-            msg = f"Wrong input '{detail_input}'. {detail_msg}"
-
+    try:
+        resp = exc.response.json()
+        detail = resp.get("detail")
+        # Default message
+        msg = "Internal error."
+
+        if isinstance(detail, str):
+            msg = detail
+        elif isinstance(detail, list):
+            detail_input = detail[0]["input"]
+            detail_msg = detail[0]["msg"]
+
+            if detail_input and detail_msg:
+                msg = f"Wrong input '{detail_input}'. {detail_msg}"
+    except:
+        # If something fails decoding the error
+        # just show the incoming error
+        msg = f"Internal error: {str(exc)}"
```

Comment:
I'm not sure if in python is a good practice to have an `except` without possible exceptions. If not, I would suggest to do:

```suggestion
    # Default message
    msg = "Internal error."
        
    try:
        resp = exc.response.json()
        detail = resp.get("detail")

        if isinstance(detail, str):
            msg = detail
        elif isinstance(detail, list):
            detail_input = detail[0]["input"]
            detail_msg = detail[0]["msg"]

            if detail_input and detail_msg:
                msg = f"Wrong input '{detail_input}'. {detail_msg}"
    except (KeyError, TypeError, ValueError) as e:
        # If something fails decoding the error
        # just show the incoming error
        msg = f"Internal error: {str(e)}"
```

Reply:
Actually it is not but I would put a plain `Exception`  there as I do not want anything masking the real error. In any case I had to work on other urgent stuff and leave this as it is. I need to fix some tests, etc. 'll retake it when I have time!